plugins/plugins.go

```diff
@@ -8,12 +8,19 @@ package plugins
 import (
 	"context"
 	"encoding/json"
+	"sync"
 
+	"github.com/open-policy-agent/opa/ast"
 	"github.com/open-policy-agent/opa/plugins/rest"
 	"github.com/open-policy-agent/opa/storage"
 	"github.com/open-policy-agent/opa/util"
 )
 
+var (
```

Comment:
Can these be moved into the manager? If so, that would be better. Avoid global state if possible.

Reply:
Done